src/wmtk/io/ParaviewWriter.cpp

```diff
@@ -84,7 +84,8 @@ ParaviewWriter::ParaviewWriter(
     for (int i = 0; i < 4; ++i) {
```

Comment:
As you are on it already anyways, please change `int` to `size_t` here.

Reply:
4 is a constant that can be an int, why size_t

Reply:
For me it's a good practice when potentially using stl accessors to prevent the casting, I always assume the cast down is very cheap/easy too

Reply:
agree, but this one is just 4 :)

Reply:
Sure but you intend to use it in a context of a `size_t`. This one here is not an issue but I have quite a list of warnings in Windows because people convert `size_t` into `long` everywhere.

So, here it is fine (does not produce a warning) but I'd prefer if we are in general more careful with integer types.